script/workLoggingSystem.js

```diff
@@ -135,10 +135,78 @@ $("#submitNewTask").click(function() {
     console.log(estimatedTime);
     console.log(status);
     console.log(category);
+    if(taskpriority(priority,difficulty,status,category,estimatedTime))
+        {
+            console.log("working");
+            createNewTask(title,des,priority,difficulty,creator,assignee,estimatedTime,status,category);
+        }
+    else
+        {
+            console.log("wrong values");
+        }
 
-    createNewTask(title,des,priority,difficulty,creator,assignee,estimatedTime,status,category);
 });
 
+function taskpriority(priority,difficulty,status,category,estimatedTime) {
+    var p=0,d=0,s=0,c=0,e=0;
+    if(priority == "High" || priority == "Medium"|| priority == "Low")
+        {   
+            $(".ch").css({"background-color": "#ffc107", "color": "black"});
+            p=1;
+        }
+    else    
+        {
+            $(".ch").css({"background-color": "red", "color": "#ffc107"});
+        }
+    if(difficulty == "High" || difficulty == "Medium"|| difficulty == "Low")
+        {   
+            $(".ch").css({"background-color": "#ffc107", "color": "black"});
+            d=1;
+        }
+    else    
+        {
+            $(".ch").css({"background-color": "red", "color": "#ffc107"});
+        }
+    if(status == "Icebox" || status == "Ready to start"|| status == "Under Progress"|| status == "Blocked"|| status == "Completed")
+        {   
+            $(".ch").css({"background-color": "#ffc107", "color": "black"});
+            s=1;
+        }
+    else    
+        {
+            $(".ch").css({"background-color": "red", "color": "#ffc107"});
+        }
```

Comment:
giving class ch to all will turn all the options red in color, where as it will be difficult to identify which one is not passing validation. Please give all of them a unique id with reasonable name and focus to change the particular id only which has not passed validation.